.github/workflows/audit.yml

```diff
@@ -43,13 +43,17 @@ jobs:
       working-directory: examples/connector/python
       run: ./build.sh
 
-    - name: Install Python destination requirements
-      working-directory: examples/destination/python
+    - name: Install Python simple destination requirements
+      working-directory: examples/destination/python/simple
       run: pip install -r requirements.txt
 
-    - name: Run Python destination build script
-      working-directory: examples/destination/python
+    - name: Run Python simple destination build script
```

Comment:
Vector db should also run a ./build.sh in order to at least check that the code will build.  Just installing the requirements is not enough